source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -0,0 +1,286 @@
+#include "extensions/filters/http/jwt_authn/authenticator.h"
+
+#include "envoy/http/async_client.h"
+
+#include "common/common/logger.h"
+#include "common/http/message_impl.h"
+#include "common/http/utility.h"
+
+// from jwt_verify library
+#include "jwt.h"
```

Comment:
Can you import (or export) these under some prefixed path?

Reply:
Done